modules/@apostrophecms/page/ui/apos/components/AposPagesManager.vue

```diff
@@ -31,11 +31,26 @@
       <AposButton
         v-if="relationshipField"
         type="primary"
-        label="Select Pages"
-        :disabled="relationshipErrors === 'min'"
+        :label="saveRelationshipLabel"
+        :disabled="(relationshipErrors === 'min') || (relationshipErrors === 'max')"
```

Comment:
what errors would not warrant disabling save? Should this just be `:disabled ="!!(relationshipErrors)"` ?